functions/util.py

```diff
@@ -67,3 +69,43 @@ class BackgroundContext(NamedTuple):
     timestamp: str  # ISO 8601
     event_type: str  # e.g., "google.pubsub.topic.publish"
     resource: str
+
+
+def get_blob_as_stream(
```

Comment:
I might be wrong, but don't we have that already like in API code? Should it maybe just import it from there?

Reply:
Don't think so! I only see functions for uploading blobs in the API code.